### ISSUE TITLE
Feature Request: Different display options

### ISSUE THREAD

**Issue by Pigothy** [hardcore-sushi/DroidFS#87]
Right now for example pictures are just shown with a tiny thumbnail on the left and the name of the picture on the right. With similar pictures it's very hard to actually find the right one since the name is going to be extremely similar for all them as well.

It would be hugely beneficial to add different display options, like the current one would be a list view, and the one I would love to have would be a thumbnail display, were you have just the thumbnails no file name, just having like maybe 2-3 thumbnails across your screen and then obviously just scrolling down till the end of your directory.

Would make it so much easier to see and find pictures.

**Comment by hardcore-sushi** [comment]
Good idea but file names should always be shown because sometime thumbnails cannot be displayed and files are not always pictures.

**Comment by hardcore-sushi** [comment]
Yes, we should allow all RecyclerViews to be displayed as a grid.

**Comment by hardcore-sushi** [comment]
I implemented it for the file explorer: 4f9aa55dfe3ed96a5b8ab321e2c869f10bf127f8
However, I don't know how to change the home page UI. The icons are all the same so I don't really find it relevant to display them in large size. I also thought of putting the name of the volume instead, but if it's too long, it won't be displayed in full. Any suggestions?

**Comment by cyanwolfg** [comment]
I don't know the difficulty of implementing it but in some applications such as contacts when there is no image to show a thumbnail is generated with the first letter of the name to differentiate between the information, maybe that could be an option to use the same design as in the file explorer but generating a different thumbnail for each volume.

Another alternative that occurs to me is to be able to choose different sizes of the interface showing in list but increasing or decreasing the size of the thumbnail and the font of the text.
